src/view/components/cpx/CpxSimulator.tsx

```diff
@@ -65,9 +62,9 @@ const sendMessage = (type: string, state: any) => {
     vscode.postMessage({ command: type, text: state });
 };
 
-class Simulator extends React.Component<any, IState> {
-    constructor(props: IMyProps) {
-        super(props);
+class Simulator extends React.Component<{}, IState> {
+    constructor() {
```

Comment:
constructor(props) {
    super(props);
}

Reply:
Don't need the type here, It'll automatically infer from your React.Component definition

Reply:
It doesn't infer it automatically since I have no implicit any rule. I can do this instead:
    constructor(props: Readonly<{}>) {
        super(props);

Reply:
sounds good. feel free to resolve this